testing/py-ycmd/py-ycmd.post-install

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Lets put this message as `README.Alpine` in `/usr/share/doc/ycmd`.  Then if people install the documentation hooks for apk it will display it, instead of cluttering install/update logs.